src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,19 +32,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
```

Comment:
Please add a comment describing the responsibility of this class.

Reply:
Done!